.puppet-lint.rc

```diff
@@ -1 +0,0 @@
-

```

Comment:
It looks like this was formerly a CRLF, but this update removed even that. It's not a completely empty file. A hex dump of this version of the file returns no bytes at all. I don't know how the CRLF got there in the first place in all the other templates and it could be that the process Erick used to fix his line endings got rid of it. I don't know if this is one of the ones he had to fix the line endings or not. But I'm thinking that if the templates ever want to put any content in here, not having a newline in here now shouldn't be an issue.